### ISSUE TITLE
Tab content accessibility docs are vague

### ISSUE THREAD

**Issue by sowasred2012** [canonical-web-and-design/vanilla-framework#3897]
https://vanillaframework.io/docs/patterns/tabs#content should be more explicit that keyboard events (specifically arrow key presses) are needed for this pattern to be accessible. The example JS includes these keyboard events, but it's not specifically called out in the copy, and easily missed.

Noticed in: https://github.com/canonical-web-and-design/ubuntu.com/pull/10061
